python/feathub/feature_views/feature_view.py

```diff
@@ -56,15 +64,19 @@ def __init__(
             # Uses table's keys as features' keys if features' keys are not specified.
             for feature in [f for f in self.get_resolved_features() if f.keys is None]:
                 feature.keys = keys
+
+        if is_unresolved:
+            timestamp_field = None
```

Comment:
What is the purpose of passing `timestamp_field` as inputs to the constructor given that we will anyway overwrite its value here?